components/resources/brave_new_tab_page_strings.grdp

```diff
@@ -69,6 +69,10 @@
     Hide top sites
   </message>
 
+  <message name="IDS_NEW_TAB_NEWS_WIDGET_TITLE" desc="Title for Brave News widget">
+    NEWS
```

Comment:
interesting, we want this capitalized?

Reply:
The designs use caps for the widget titles, although perhaps case conversion could be done in CSS.

Reply:
either is fine, just wanted to double check :smile: